src/routes/changelog/+page.svelte

```
@@ -1,29 +1,669 @@
+									<div
+										class="rounded-xl border border-base-300/80 bg-base-100/70 p-2 mb-3 w-fit"
+									>
+										<Icon size={16} />
+									</div>
+									<h3 class="text-sm font-semibold">{highlight.title}</h3>
+									<p class="mt-1 text-xs text-base-content/55 leading-relaxed">
+										{highlight.description}
+									</p>
+									{#if highlight.href}
+										<a
+											href={highlight.href}
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)